__tests__/commands/install.js

```diff
@@ -520,13 +520,23 @@ test.concurrent(
 );
 
 test.concurrent('install should hoist nested bin scripts', (): Promise<void> => {
-  return runInstall({}, 'install-nested-bin', async (config) => {
+  return runInstall({binLinks: true}, 'install-nested-bin', async (config) => {
     const binScripts = await fs.walk(path.join(config.cwd, 'node_modules', '.bin'));
-    assert.equal(binScripts.length, 10);
+    // need to double the amount as windows makes 2 entries for each dependency
+    // so for below, there would be an entry for eslint and eslint.cmd on win32
+    const amount = process.platform === 'win32' ? 20 : 10;
```

Comment:
Good catch! Unfortunately we do need both on Windows - the extension-less version is used with bash-like shells (eg. "Bash on Ubuntu on Windows", Git Bash, Babun, Cygwin, etc) while the .cmd version is used by regular terminals (cmd.exe, Cmder/ConEmu, PowerShell, etc).